app/jobs/bulk_action_jobs.rb

```diff
@@ -272,4 +272,14 @@ def perform(collection_id, overwrite = true, save_field = nil)
       [successes, errors]
     end
   end
+
+  class ReturnCheckouts < ActiveJob::Base
+    queue_as :bulk_return_checkouts
```

Comment:
Do you think this needs a new queue?  I'm guessing it won't happen very often and thus would probably be fine on the default queue.  If you think it would be better being on its own then you need to add it to `config/sidekiq.yml` to let sidekiq know to run them.

Reply:
I think it is fine as part of the default queue for now. I would expect it will be a fairly rare thing to trigger in real world use and we can always add a queue later if it becomes problematic for an implementer.